base/data/gm4/tags/blocks/no_collision.json

```diff
@@ -0,0 +1,73 @@
+{
+  "values": [
+    "#gm4:air",
+    "#gm4:water",
+    "#minecraft:banners",
+    "#minecraft:buttons",
+    "#minecraft:corals",
+    "#minecraft:crops",
+    "#minecraft:flowers",
+    "#minecraft:portals",
+    "#minecraft:pressure_plates",
+    "#minecraft:rails",
+    "#minecraft:saplings",
+    "#minecraft:signs",
+    "#minecraft:wall_corals",
+    "minecraft:attached_melon_stem",
+    "minecraft:attached_pumpkin_stem",
+    "minecraft:beetroots",
+    "minecraft:brown_mushroom",
+    "minecraft:carrots",
```

Comment:
The following is already part of #minecraft:crops and should be removed for redundancy:
    "minecraft:beetroots",
    "minecraft:carrots",
    "minecraft:potatoes",
    "minecraft:wheat",
    "minecraft:melon_stem",
    "minecraft:pumpkin_stem"